ArcadiaTeamsBot/appsettings.json

```diff
@@ -10,5 +10,10 @@
   "ServiceDesk": {
     "ApiUrl": "",
     "ApiKey": ""
+  },
+  "ServiceDeskRequestsMapping": {
+    "Second field": "Number",

```

Comment:
It should be request type specific, not global.

Reply:
Changed appsettings.json structure.